integrationtests/wildfly-modules/src/test/resources/arquillian.xml

```diff
@@ -11,10 +11,13 @@
             <configuration>
                 <property name="jbossHome">${basedir}/target/node1/wildfly-${version.org.wildfly}</property>
                 <property name="javaVmArguments">-Djboss.socket.binding.port-offset=100 -Xmx512m -Djava.net.preferIPv4Stack=true
-                    -Djgroups.bind_addr=127.0.0.1 -Djgroups.join_timeout=2000
+                     -Djgroups.bind_addr=127.0.0.1 -Djgroups.join_timeout=2000
                 </property>
                 <!-- To debug the Arquillian managed application server: -->
-                <!-- property name="javaVmArguments">-Xrunjdwp:transport=dt_socket,address=8000,server=y,suspend=y -Xmx512m -Dorg.jboss.remoting-jmx.timeout=300  -Djava.net.preferIPv4Stack=true -Djgroups.bind_addr=127.0.0.1</property-->
+                <!--<property name="javaVmArguments">-Xrunjdwp:transport=dt_socket,address=8000,server=n,suspend=y -Dorg.jboss.remoting-jmx.timeout=300-->
+                    <!-- -Djboss.socket.binding.port-offset=100 -Xmx512m -Djava.net.preferIPv4Stack=true-->
```

Comment:
I really want to keep this in because the existing commented `javaVmArguments` were incomplete, and it was frustrating when I enabled debugging and the test wouldn't run.